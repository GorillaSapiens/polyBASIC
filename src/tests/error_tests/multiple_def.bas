10 DEF a(x) 1
20 DEF a(x) 2
