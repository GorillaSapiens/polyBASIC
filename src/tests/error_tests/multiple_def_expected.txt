ERROR: @2:4 FUNCTION DEFINITION MULTIPLY DEFINED ❰a❱ @1:4
